### ISSUE TITLE
Space between menu button and submenu in IE

### ISSUE THREAD

**Issue by andrei-ivanov** [primefaces/primefaces#1071]
Hi,
In IE (tried 9 and 11), there is a space between the menu (the one with "Actions" label) and the dropdown containing the items that shows up on mouse over.
This causes the very annoying issue that moving the mouse towards the menu items will reach that space causing the submenu to dissapear. The only way to make it is to move the mouse very fast over that gap.

As far as I see this is caused by the submenu being smaller on IE (height 16px) than on Firefox (height 17px).

``` xml
<p:menubar>
    <p:submenu label="Actions" icon="ui-icon-wrench">
        <p:menuitem value="Edit User" url="UserDetails.jsf?edit=true" />
        <p:menuitem value="Add Role" url="RoleDetails.jsf?add=true" />
        <p:menuitem value="Add Team" url="TeamDetails.jsf?add=true" />
    </p:submenu>
</p:menubar>
```

Using version 5.3.0.

**Comment by sudheerj** [comment]
Which menu component you are trying on? What changes we have to do replicate the issue on showcase example?

**Comment by andrei-ivanov** [comment]
I've updated the description with more details.

**Comment by andrei-ivanov** [comment]
We're also using the Flick theme, version 1.0.10.
Trying the official showcase page with the Flick theme, I can't reproduce the issue.
http://www.primefaces.org/showcase/ui/menu/menubar.xhtml

But looking with the browser DOM inspector, I don't see any custom styles getting applied to the elements, only PF styles and theme styles.

**Comment by melloware** [comment]
@mertsincan I reported this issue as a PRO user and you fixed it in 5.3.18, 6.0.5 and 6.1
